src/order_type.h

```diff
@@ -25,7 +25,7 @@ static const VehicleOrderID INVALID_VEH_ORDER_ID = 0xFF;
 static const VehicleOrderID MAX_VEH_ORDER_ID     = INVALID_VEH_ORDER_ID - 1;
 
 /** Invalid order (sentinel) */
-static const OrderID INVALID_ORDER = 0xFFFF;
+static const OrderID INVALID_ORDER = 0xFFFFFF;
```

Comment:
Doesn't this change require conversion on load ?

Reply:
Orders are stored as references which already does the magic. See 1fa98e169529b070c19a6bc0cff311ebc993b468 for a similar change, 9 years ago :-)

There would also be an issue if OrderID was passed in commands, but it seems VehicleOrderIDs are used instead.